system/Debug/Iterator.php

```diff
@@ -39,6 +39,12 @@ class Iterator
      * things to happen during the test.
      *
      * @return $this
+     *
+     * @template closureReturn
+     *
+     * @phpstan-param Closure(): closureReturn $closure
```

Comment:
What does using the template accomplish if it isn't referenced elsewhere? I think this could be moved to the class level and added to the `$tests` property array, but I'm still not sure what `closureReturn` helps with.

Reply:
I think I see, you are trying to specify that the Closure takes no arguments and may have a return value. Why not `Closure(): mixed`?

Reply:
Hi, @MGatner 
Do you mean this way?
`@param Closure(): mixed $closure`

Reply:
done.